types/tslint.json

```diff
@@ -0,0 +1 @@
+{ "extends": "dtslint/dtslint.json" }
```

Comment:
This file is not _required_ unless we want linting capabilities _within_ this repo.